README.md

```diff
@@ -8,8 +8,8 @@ Learn how to engage with the Kubernetes community on the [community page](http:/
 
 You can reach the maintainers of this project at:
 
-- [Slack](http://slack.k8s.io/)
-- [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-dev)
+- [Slack](http://slack.k8s.io/messages/cluster-api)
```

Comment:
This url does not seem to work for me @Lion-Wei

Reply:
oops, seems like I can't modify on this branch. So I'll close this pr and open a new one.